spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -18,19 +18,25 @@
 
```

Comment:
As far as documentation goes, I agree that we could provide a new page in [4. Computing shape metrics for PAM50-registered data](https://spinalcordtoolbox.com/en/latest/user_section/tutorials/shape-metric-computation.html).

Something like "4.5 CSA (Region based on distance from PMJ)"